Logstash/logstash-input-azureblob/lib/logstash/inputs/azureblob.rb

```diff
@@ -109,26 +109,35 @@ class LogStash::Inputs::LogstashInputAzureblob < LogStash::Inputs::Base
   # Constant of max integer
   MAX = 2 ** ([42].pack('i').size * 16 -2 ) -1
 
+  # 4MB
+  MAX_BUFFER_SIZE = 4 * 1024 * 1024 
+
+
   public
   def register
+    user_agent = "logstash-input-azureblob"
+    user_agent << "/" << Gem.latest_spec_for("logstash-input-azureblob").version.to_s
```

Comment:
Make sure there is an easy way to query this across all versions, for example, something like prefix(user_agent) == "logstash-input-azureblob"

Reply:
I don't understand. The current user agent is "logstash-input-azureblob/0.9.11"

Reply:
Imagine you have a database storing these data with rows storing them as
logstash-input-azureblob/0.9.8
logstash-input-azureblob/0.9.9
logstash-input-azureblob/0.9.10
...

The question is, can you write a query that returns count for all logstash version, irrespective of the specific plugin versions.